### ISSUE TITLE
Accessibility Issues

### ISSUE THREAD

**Issue by samjonester** [testdouble/double-takes#78]
- Navigation hamburger and close are not accessible to keyboard users
- Hamburger and close buttons should have aria-label, since they don't have text.
- Aria-controls should be used on the hamburger to link it to the side nav http://tink.uk/using-the-aria-controls-attribute/
- aria-expanded should probably be used to denote the toggle state of the nav bar, but that would require JS.
- After opening side nav, links in the side nav should be next in tab index, currently they are last on the page due to html ordering.

**Comment by jasonkarns** [comment]
Skip links would be useful as well.

**Comment by searls** [comment]
Since you both necessarily know a whole lot more than me, could one of you take a stab at whatever remediations can be made without JavaScript, but in the testdouble/site first? (This repo is a downstream dependency of that one despite being committed into it)

**Comment by samjonester** [comment]
Definitly! @JoshTGreenwood Was also interested in learning about accessibility, and we'll be pairing on it. Also, yes, we look at testdouble/site first :)
